docs/source/demos/cost_benefit_matrix.ipynb

```diff
@@ -74,9 +74,21 @@
     "from evalml.demos.churn import load_churn\n",
     "from evalml.preprocessing import split_data\n",
     "X, y = load_churn()\n",
+    "X = X.set_types({'PaymentMethod':'Categorical', 'Contract': 'Categorical'}) # Update data types Woodwork did not correctly infer\n",
```

Comment:
Woodwork inferred these as natural language, causing a text featurizer to be added and the runtime to significantly increase so manually updating here.